src/styles/components/layout/Verso/VersoContent/VersoContentOffer/_VersoContentOffer.scss

```
@@ -1,6 +1,21 @@
 .verso-info {
   padding: $page-padding;
 
+  .duo {
+    span:first-child  {
+      border-radius: 10px;
+      font-size: 12px;
```

```suggestion
      font-size: rem(12px);
```
Mieux vaut mettre tout ce qui est en `px` de cette façon pour être accessible.